addons/project/tests/test_project_report.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from odoo.tests import tagged, TransactionCase
+from odoo import Command
+from odoo.tests.common import new_test_user
+from odoo.tools.float_utils import float_compare
+from odoo.addons.rating.models.rating_data import RATING_LIMIT_MIN
+from odoo.addons.mail.tests.common import MockEmail
+
+@tagged('post_install', '-at_install')
+class TestProjectReport(TransactionCase, MockEmail):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Partner',
+        })
+        cls.project = cls.env['project.project'].create({
+            'name': 'project'
+        })
+        cls.user = new_test_user(cls.env, login='test user')
+        cls.task = cls.env['project.task'].create({
+            'name': 'Fsm task',
+            'project_id': cls.project.id,
+            'partner_id': cls.partner.id,
+            'user_ids': [Command.set([cls.user.id])],
+        })
+
+    def test_avg_rating_measure(self):
+        def avg(_sum, _len):
+            if _len > 0:
+                return _sum/_len
+            return -1            
+
+        ratings = [5, 4]
+        feedbacks = ['Goog Job', 'Good Job but with one day delay']
+        for i in range(2):
+            rating = self.env['rating.rating'].create({
+                'res_model_id': self.env['ir.model']._get('project.task').id,
```

Comment:
i tried this and the result was wrong 
res_model = False instead of 'project.task' 
and res_model_id = ir.model() instead of ir.model(id) 
so i will keep my version